corehq/apps/reports/tasks.py

```diff
@@ -370,7 +370,7 @@ def _get_export_properties(export):
     return properties
 
 
-def find_question_id(form, value):
+def find_question_id(form, value, last_path=False):
```

Comment:
Thanks for this feedback. I went with Daniel's suggestion for `use_basename` since that is the method name. I did some rearranging/renaming to hopefully make this a bit cleaner.